pinot-common/src/main/java/org/apache/pinot/common/metrics/BrokerMeter.java

```diff
@@ -63,6 +63,7 @@
   DOCUMENTS_SCANNED("documents", false),
   ENTRIES_SCANNED_IN_FILTER("documents", false),
   ENTRIES_SCANNED_POST_FILTER("documents", false),
+  REQUEST_SIZE("requestSize", false),
```

Comment:
This should be a gauge, not a meter. 
A meter is a counter, whereas a gauge is an actual value at any point in time. 
Add the metric to BrokerGauge

Reply:
done.